wallaby.js

```diff
@@ -1,14 +1,23 @@
-module.exports = function (wallaby) {
+module.exports = function(wallaby) {
     return {
+        env: {
+            type: 'node',
+            runner: 'node',
+        },
         files: [
-            {pattern: 'scripts/**/*.js'},
+            'scripts/**/*.js',
+            '!scripts/**/*.spec.js',
```

Comment:
Because your tests are side-by-side, the pattern above will also match the tests in addition to the source files. Therefore, we explicitly exclude tests files; as they will be added in the `tests: []` property value.